### DISCUSSION TITLE
Update Node red - no installation found

### DISCUSSION THREAD

**Discussion by TadFocus** [tteck/Proxmox#3815] [Help Needed] [answered]
Hi all, I need advice on updating Node-RED without losing all data and flows. I keep getting this error. Thank you ;)
![Snímek obrazovky 2024-10-02 062657](https://github.com/user-attachments/assets/c3ef7f1d-0e2f-4f61-9f35-9574e99fa915)

**Comment by tteck** [comment]
Looks as if Node-Red wasn't installed using the script from this repository.

**Reply by TadFocus** [reply, marked as answer]
I tried reinstalling it and now it works, thanks for the nudge.
